tests/functional/test_change_email_address.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+q_email = 'some.one@digital.cabinet-office.gov.uk'
+
+email_depts = [('some.one@cabinetoffice.gov.uk',
+                'Cabinet Office')]
+
+
+class When_on_change_email(object):
```

Comment:
This doesn't read very clearly. `When_changing_email_address` or `When_on_change_email_page` would be better.